netty/src/test/java/io/grpc/netty/WriteQueueTest.java

```diff
@@ -61,14 +62,16 @@
 @RunWith(JUnit4.class)
 public class WriteQueueTest {
 
+  private final Object lock = new Object();
+
   @Mock
   public Channel channel;
 
   @Mock
   public ChannelPromise promise;
 
-  private long flushCalledNanos;
   private long writeCalledNanos;
+  private long flushCalledNanos = writeCalledNanos + 1;
```

Comment:
There is a `scheduleFlush` method, that does just that. It is used when some other code is handling the write (because it is already on the event loop) and just needs to make sure a flush occurs.